content/en/docs/tasks/traffic-management/request-timeouts/index.md

```diff
@@ -29,7 +29,7 @@ timeout to half a second.
 To see its effect, however, you also introduce an artificial 2 second delay in calls
 to the `ratings` service.
 
-1.  Route requests to v2 of the `reviews` service, i.e., a version that calls the `ratings` service:
+1)  Route requests to v2 of the `reviews` service, i.e., a version that calls the `ratings` service:
```

Comment:
In my understanding, `1.` is correct before `{{< tabset>}}`, and 2), 3)... will be used after it in sequence. Need a confirmation :)

Reply:
Thanks for the professional explanation, please help to confirm, if yes, I will close this PR.

Reply:
It is fine as is. The ) of swapped to `.` in the final render.

Reply:
Thanks, I will close this change.